accessibility_monitoring_platform/apps/cases/models.py

```diff
@@ -849,11 +850,17 @@ def psb_response(self) -> bool:
 
     @property
     def audit(self):
-        return self.audit_case.filter(is_deleted=False).first()  # type: ignore
+        try:
+            return self.audit_case  # type: ignore
+        except ObjectDoesNotExist:
+            return None
 
     @property
     def report(self):
-        return self.report_case.filter(is_deleted=False).first()  # type: ignore
+        try:
+            return self.report_case  # type: ignore
+        except ObjectDoesNotExist:
+            return None
```

Comment:
What happens when it returns None? Does it produce a 500 error?

Reply:
@richardbored No the `ObjectDoesNotExist` exception would have returned a 500 error. This returns either the report or `None` which is what the surrounding logic expects.